cms/envs/aws.py

```diff
@@ -365,6 +365,10 @@
 PROCTORING_BACKEND_PROVIDER = AUTH_TOKENS.get("PROCTORING_BACKEND_PROVIDER", PROCTORING_BACKEND_PROVIDER)
 PROCTORING_SETTINGS = ENV_TOKENS.get("PROCTORING_SETTINGS", PROCTORING_SETTINGS)
 
+################# MICROSITE ####################
+
```

Comment:
Please include a comment here describing the purpose of these settings, and which settings are available for configuration in this location.